source/solutions-library/retail-as-you-type-suggestions.txt

```diff
@@ -265,18 +249,19 @@ in turn surface all matching items.
 
 Visit the `GitHub repo <https://github.com/mongodb-developer/atlas-search-as-you-type-suggest>`__ to this solution. 
 
-Key Considerations
-------------------
+Key Learnings
+-------------
 
-Model suggestible entities as documents with a specialized index
-configuration: This could be done as described above in a separate
-collection containing entities from any source. Or, if your main
-collection models all suggestible entities as top-level documents
-already, an index can be created or an existing one augmented to use the
-index configuration techniques described here.
+- **Model suggestible entities as documents with a specialized index
+  configuration:** This could be done as described above in a separate
+  collection containing entities from any source. 
 
-Craft clever queries: Leverage the index structure, generating rich and
-nuanced queries to match entities and rank suggestions as desired.
+- **Index Creation:** If your main collection models all suggestible entities as
+  top-level documents already, an index can be created or an existing one
+  augmented to use the index configuration techniques described here.
+
+- **Craft clever queries:** Leverage the index structure, generating rich and
+  nuanced queries to match entities and rank suggestions as desired.
 
 Technology and Products
 -----------------------
```

Comment:
we've generally been removing the technology and products sections since the products are mentioned at the beginning of the page.